WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/personalize/PersonalizeCardViewHolder.kt

```diff
@@ -0,0 +1,30 @@
+package org.wordpress.android.ui.mysite.cards.personalize
+
+import android.view.ViewGroup
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import org.wordpress.android.databinding.PersonalizeCardBinding
+import org.wordpress.android.ui.compose.theme.AppTheme
+import org.wordpress.android.ui.mysite.MySiteCardAndItem
+import org.wordpress.android.ui.mysite.cards.dashboard.CardViewHolder
+import org.wordpress.android.util.extensions.viewBinding
+
+class PersonalizeCardViewHolder(parent: ViewGroup) :
+    CardViewHolder<PersonalizeCardBinding>(parent.viewBinding(PersonalizeCardBinding::inflate)) {
+    fun bind(cardModel: MySiteCardAndItem.Card.PersonalizeCardModel) = with(binding) {
+        this.personalizeCard.apply {
+            setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnDetachedFromWindowOrReleasedFromPool)
```

Comment:
The default value of viewCompositionStrategy is `DisposeOnDetachedFromWindowOrReleasedFromPool`. I don't think this is necessary. 

Check this [comment](https://github.com/wordpress-mobile/WordPress-Android/pull/19130#issuecomment-1705078083)

Reply:
👋 @AjeshRPai . I didn't not realize this, thank you for the reference. I removed the strategy.